### ISSUE TITLE
`Bug` chips status color media player

### ISSUE THREAD

**Issue by Mars2020-Github** [Blackymas/NSPanel_HA_Blueprint#1716]
### TFT Version

4.2.4

### ESPHome Version

4.2.4

### Blueprint Version

4.2.4

### Panel Model

NSPanel EU Model

### What is the bug?

Looks like an old bug returned in the latest version…. The color of the media icons (Spotify etc) is not updated real-time on the homescreen. Only if you refresh the homescreen, the icon color (status) is updated.

### Steps to Reproduce

Include media player in chips.
Turn on or off mediaplayer (Spotify or Marantz etc).
Chip color stay’s “on” and color only goed back to “off state” if screen us refreshed.

### Your panel's YAML

```yaml
chip01_icon: ""
    home_custom_button01: media_player.spotify_karin_versteegh
    home_custom_button01_icon: mdi:spotify
    home_custom_button02_icon: mdi:audio-video
    home_custom_button03: media_player.spotify_marulaver
    home_custom_button03_icon: mdi:spotify
    home_value01_label_color:
```

**Comment by edwardtfn** [comment]
Media player is not yet supported on Chips, so I believe you have issues with custom buttons, is that right?

**Comment by edwardtfn** [comment]
I have a fix for this in `dev` and it will be included in the next release.

**Comment by edwardtfn** [comment]
We just released a patch with solves this: [v4.2.5 - Celebrating 1000 Stars with Enhanced Functionality and UI Improvements](https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.2.5)
Could you please update your system and let me know if the issue was solved?
